packages/blocto-sdk/src/providers/ethereum.ts

```diff
@@ -443,6 +444,7 @@ export default class EthereumProvider
       case 'wallet_addEthereumChain': {
         return this.loadSwitchableNetwork(payload?.params || []);
       }
+      case 'eth_blockNumber':
```

Comment:
uniswap makes periodic calls to `eth_blockNumber`, adding `eth_blockNumber` to avoid falling into the default switch block that requires user to be connected (L527)